### PR TITLE
Mini aod dev 76 x

### PR DESCRIPTION
**Pull request by Nabarun21**
Re-Add the eVetoMVAIsoVtx and eVetoMVAIso branches with the ID changed .
https://github.com/uwcms/FinalStateAnalysis/issues/478
Add unsigned long to variable types in cython proxy maker. make_cython_proxy
https://github.com/uwcms/FinalStateAnalysis/issues/483

### CODE REVIEW

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by isobelojalvo** [comment]
Bucky, shut up!

**Comment by dntaylor** [comment]
add to whitelist

File: NtupleTools/python/templates/cleaning.py [hunk 1]
```diff
@@ -33,7 +33,8 @@
     #ELECTRON VETOS
     #eVetoMVAIsoVtx = 'vetoElectrons(0.4, "pt > 10 & abs(eta) < 2.5 & userInt(\'mvaidwp\') > 0.5 & ((userIso(0) + max(userIso(1) + neutralHadronIso - 0.5*userIso(2), 0))/pt) < 0.3 & userFloat(\'dz\') < 0.2").size()',
     #eVetoMVAIso = 'vetoElectrons(0.4, "pt > 10 & abs(eta) < 2.5 & userInt(\'mvaidwp\') > 0.5 & (userIso(0) + max(userIso(1) + neutralHadronIso - 0.5*userIso(2), 0))/pt < 0.3").size()',
-    
+    eVetoMVAIsoVtx = 'vetoElectrons(0.4, "pt > 10 & abs(eta) < 2.5 & userFloat(\'MVANonTrigWP90\') > 0.5 & ((userIso(0) + max(userIso(1) + neutralHadronIso - 0.5*userIso(2), 0))/pt) < 0.3 & userFloat(\'dz\') < 0.2").size()',
```
Comment (nwoods) [inline NtupleTools/python/templates/cleaning.py:36]: In my opinion, this kind of thing should be moved to an analysis-specific parameter file. An example may be found [here](https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_76X/NtupleTools/python/parameters/wz.py#L66).

File: NtupleTools/python/templates/cleaning.py [hunk 2]
```diff
@@ -33,7 +33,8 @@
...
+eVetoMVAIso = 'vetoElectrons(0.4, "pt > 10 & abs(eta) < 2.5 & userFloat(\'MVANonTrigWP90\') > 0.5 & (userIso(0) + max(userIso(1) + neutralHadronIso - 0.5*userIso(2), 0))/pt < 0.3").size()',
```
Comment (nwoods) [inline NtupleTools/python/templates/cleaning.py]: Please fix the indentation here. I'm a little surprised Python didn't complain about this when you tested the PR.

File: NtupleTools/python/templates/trigger.py [hunk 3]
```diff
@@ -54,10 +54,6 @@
         name='singleE_leg2', 
         paths=r'HLT_Ele12_CaloIdL_TrackIdL_IsoVL_v\\d+'
         ),
-    _trig_template.replace(
```
Comment (nwoods) [inline NtupleTools/python/templates/trigger.py:57]: Why is this being removed? As far as I know, @dntaylor and others use it.

Comment (Nabarun21) [inline NtupleTools/python/templates/trigger.py:57]: I didn't remove it. I had added it to my release before somebody else added it to the main branch list at a different position. So when  pulled from the main release it got added twice. I just removed one of the double additions. It's still there.

Comment (nwoods) [inline NtupleTools/python/templates/trigger.py:57]: Ah, I see it now. No problem here then.

File: NtupleTools/python/templates/cleaning.py (hunk 2 above)
Comment (Nabarun21) [inline NtupleTools/python/templates/cleaning.py]: Hi
So , I fixed the indentation, as you can see below. The checks are now failing. I am very new to github and a little confused. Do i have to submit another pull request?

**Comment by dntaylor** [comment]
The tests are failing because of no disk space. However, I just cleaned it out and it filled back up again. I'm working on it.

**Comment by dntaylor** [comment]
Okay, reran the checks after fixing jenkins. It is okay, merging.

**Pull request merged**